Data/IntSet/Internal.hs

```diff
@@ -653,6 +654,47 @@ isSubsetOf (Tip _ _) Nil = False
 isSubsetOf Nil _         = True
 
 
+{--------------------------------------------------------------------
+  Disjoint
+--------------------------------------------------------------------}
+-- | /O(n+m)/. Check whether two sets are disjoint (i.e. their intersection
+--   is empty).
```

Comment:
Can you add some example invocations of the function and an `@since 0.5.11` annotation?